fishtest/fishtest/views.py

```diff
@@ -558,6 +578,22 @@ def del_tasks(run):
     del run['tasks']
   return run
 
+
+def update_nets(request, run_id, data):
+  if data['base_tag'] == 'master':
+    global MASTER_NET
+    MASTER_NET = get_net('master', data['tests_repo'])
```

Comment:
Does this take care that this 'master' branch / commit is also part of the official stockfish, i.e. I could push a net to my master branch on a private repo and start a test, but that wouldn't make it official. The test approval page shows if base is in master or not, so much have some logic there.

Reply:
`validate_form()` checks for equal signature, but also has ` data['base_same_as_master']` so I will use that instead.
Thanks for this feedback!

Reply:
btw, I now see you use MASTER_NET as a global, i was rather thinking that any net that ever has been in the master repository would be marked in the database (i.e. db['is_master'] = db['is_master'] or data['base_same_as_master']). In that way we could mark all nets that have been 'default net' in the history of development differently in the list. The current default in master will probably always be the one with the most recent submission date.

Reply:
Ok, changed that...